scripts/DDL.sql

```diff
@@ -1,48 +1,75 @@
+CREATE SCHEMA jobs;
 
-CREATE TABLE users(
+CREATE TABLE jobs.users(
 	upn text PRIMARY KEY,
 	display_name text
 );
 
-CREATE TABLE tags(
+CREATE TABLE jobs.tags(
 	id serial PRIMARY KEY,
 	name text
 );
 
-CREATE TABLE roles(
+CREATE TABLE jobs.roles(
 	id serial PRIMARY KEY,
 	name text
 );
 
-CREATE TABLE standards(
+CREATE TABLE jobs.standards(
 	id serial PRIMARY KEY,
 	name text
 );
 
-CREATE TABLE advertisements(
+CREATE TABLE jobs.units(
 	id serial PRIMARY KEY,
-	role_id int REFERENCES roles(id) ON DELETE CASCADE,
-	unit text,
-	branch text,
-	department text,
+	name text UNIQUE
```

Comment:
שיניתי את דעתי... כאן אין חשיבות לשנות של `מדורים/ענפים/יחידות`

כי עשויים להיות 2 מדורי Devops בצבא...

ואז יש להם את אותו שם. העיקר שיהיה ID שונה 👍

Reply:
So I removed the uniqe here